scripts/setup_macos.sh

```diff
@@ -192,29 +194,21 @@ if [[ "${_arg_modify_bashrc}" = on ]]; then
   echo -e "\n# Added by the garage installer" >> "${BASH_PROF}"
 fi
 
-# Set up MuJoCo 1.5 (for gym)
-if [[ ! -d "${HOME}/.mujoco/mjpro150" ]]; then
-  mkdir -p "${HOME}"/.mujoco
-  MUJOCO_ZIP="$(mktemp -d)/mujoco.zip"
-  wget https://www.roboti.us/download/mjpro150_osx.zip -O "${MUJOCO_ZIP}"
-  unzip -u "${MUJOCO_ZIP}" -d "${HOME}"/.mujoco
-else
-  print_warning "MuJoCo is already installed"
-fi
-# Set up MuJoCo 2.0 (for dm_control)
+# Set up MuJoCo 2.0 (for gym and dm_control)
```

Comment:
line 162 `xcode-select --install` fails if command line tools are already installed.

Reply:
i think you can use the same `set +e`/`set -e` trick used during brew install to fix this